programs/server/dashboard.html

```diff
@@ -813,91 +813,101 @@
     let edit_buttons = document.createElement('div');
     edit_buttons.className = 'chart-buttons';
 
-    let move = document.createElement('a');
-    let move_text = document.createTextNode('✥');
-    move.appendChild(move_text);
-
-    let is_dragging = false;
-    move.addEventListener('mousedown', e => {
-        const idx = getCurrentIndex();
-        is_dragging = true;
-        chart.className = 'chart chart-moving';
-
-        let offset_x = e.clientX;
-        let offset_y = e.clientY;
-
-        let displace_idx = null;
-        let displace_chart = null;
-
-        function mouseup(e) {
-            is_dragging = false;
-            chart.className = 'chart';
-            chart.style.left = null;
-            chart.style.top = null;
-
-            if (displace_idx !== null) {
-                const elem = queries[idx];
-                queries.splice(idx, 1);
-                queries.splice(displace_idx, 0, elem);
-
-                displace_chart.className = 'chart';
-                drawAll();
-            }
-        }
-
-        function mousemove(e) {
-            if (!is_dragging) {
-                document.body.removeEventListener('mousemove', mousemove);
-                document.body.removeEventListener('mouseup', mouseup);
-                return;
+    // dragging and maximizing both be added to chart only, when there are more than 1 chart
+    
+    if(charts.getElementsByClassName("chart").length != 1) {
```

Comment:
This is suboptimal, we can replace with `queries.length`.

Reply:
okay